app/src/test/java/org/simple/clinic/editpatient/EditPatientEffectHandlerTest.kt

```diff
@@ -64,14 +64,15 @@ class EditPatientEffectHandlerTest {
       identifier = Identifier(value = "1234567890abcd", type = BangladeshNationalId)
   )
 
+  private val india = TestData.country(isoCountryCode = Country.INDIA)
+  private val bangladesh = TestData.country(isoCountryCode = Country.BANGLADESH)
```

Comment:
We have a effect handler with country set to Bangladesh in this test, shouldn't we construct a new `InputFieldsFactory` for that?

This is the test
`adding an id to an empty alternative id should create a new Business id if country has alternative id`

Reply:
Good catch, will add this.

Reply:
Actually, this is not necessary because in a later commit in this feature, I have changed all the input field factories in tests to use the one for Bangladesh because the Indian one has become more complex.